.github/workflows/scala-steward.yml

```
@@ -1,6 +1,6 @@
 on:
   schedule:
-    - cron: '0 0 1,15 * *'
+    - cron: "0 0 * * 1, 5" # every Monday and Friday
```

I set Monday and Friday randomly (maybe we can run 3 times a week?)

---

I think 2 is fine :+1:

---

```suggestion
    - cron: "0 0 * * 1,5" # every Monday and Friday
```
I think there might an error and no space is allowed there?